doc/tstool.8

```diff
@@ -0,0 +1,64 @@
+.Dd July 13, 2024
+.Os Linux
+.Dt TSTOOL 8 SMM
+.Sh NAME
+.Nm tstool
+.Nd Timestamping control utility
+.Sh SYNOPSIS
+.Nm
+.Op Fl -help
+.Nm
+clock list
+.Nm
+clock info
+.Ar clock
+.Nm
+clock get
+.Ar clock
+.Nm
+clock step
+.Ar clock
+.Ar offset
+.Nm
+clock slew
+.Ar clock
+.Ar ppb
+.Nm
+clock set_to
+.Ar clock1
+.Ar clock2
+.Nm
+clock diff
+.Ar clock1
+.Ar clock2
+.Nm
+interface list
+.Nm
+interface info
+.Ar interface
+.Nm
+interface set_ts
+.Ar interface
+.Ar tx-type
+.Ar rx-filter
+.Sh DESCRIPTION
+Examine and adjust PTP Hardware Clock devices, system clock and interface timestamping settings.
+.Pp
+This utility uses the mechanisms available to sfptpd to manipulate precision clocks on the system.
+.Pp
+Clocks may be specified by phc number, interface name or 'system'.
+.Sh BUGS
+Please raise bug reports at:
+.Aq https://github.com/Xilinx-CNS/sfptpd/issues
+.Pp
+Support for users of AMD Solarflare NICs is available from:
```

Comment:
Will tstool operate on Solarflare NICs?

Reply:
> Will tstool operate on Solarflare NICs?

Yes, absolutely!

It wouldn't be needed for de-duplication but it should cover the use cases for `phc_ctl` without having to install a second alternative time sync daemon package that otherwise isn't needed. And hopefully be better than it in some ways.